src/main/java/net/syscon/elite/service/CourtHearingCancellationService.java

```diff
@@ -0,0 +1,34 @@
+package net.syscon.elite.service;
+
+import lombok.extern.slf4j.Slf4j;
+import net.syscon.elite.core.HasWriteScope;
+import net.syscon.elite.repository.jpa.repository.CourtEventRepository;
+import net.syscon.elite.security.VerifyBookingAccess;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.validation.annotation.Validated;
+
+@Service
+@Transactional(readOnly = true)
+@Validated
+@Slf4j
+public class CourtHearingCancellationService {
```

Comment:
I realise this is refers to **cancellation** but we are actually deleting.  Happy to rename...

Reply:
I think it's fine, in future cancellation might switch to e.g. a change of status but consumers of the service shouldn't care.